frontend/src/Components/loginForm.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import './login.css';
+
+export const LoginForm = () => {
+    const [email, setEmail] = useState<string>('');
+    const [password, setPassword] = useState<string>('');
+    
+    const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        try {
+          const response = await axios.post("http://localhost:8000/login/password", {
+            email,
+            password,
+          });
+          console.log(response.data)
```

Comment:
No change just yet but thinking in the future when we get a successful response, what should happen? 

We may want to add a TODO to document that

Reply:
Would it be that we redirect the user to a new page. Like maybe the landing page, their profile, or someplace else?

Reply:
Yup! Exactly, I don't know if we have a page in mind yet so it might be redirect to whatever page they were trying to access as well. 

I.e. trying to access their friend list would redirect them to login and then redirect back to the friend list page.

Reply:
yeah +1 to kevin, let's add a todo here at the very least to document that we still need to fix this

Reply:
Added!